packages/gatsby-cli/src/reporter/index.js

```diff
@@ -123,42 +219,104 @@ const reporter: Reporter = {
    * @returns {ActivityTracker} The activity tracker.
    */
   activityTimer(
-    name: string,
+    text: string,
     activityArgs: ActivityArgs = {}
   ): ActivityTracker {
-    const { parentSpan } = activityArgs
+    let { parentSpan, id } = activityArgs
```

Comment:
Should this be a const?
```suggestion
    const { parentSpan, id } = activityArgs
```

Reply:
Because `id` is not set on all activities and few lines below I do fallback to use `text` as `id` if it's not set. But this can be cleaned up. I'll look into this

Reply:
oh I probably just missed reassignment of id, no big deal 👍